code/datalab-app/package.json

```diff
@@ -61,13 +61,12 @@
     "jest": "24.7.1",
     "jest-enzyme": "7.0.2",
     "lint-staged": "3.4.1",
-    "nodemon": "1.11.0",
+    "nodemon": "1.19.1",
     "react-scripts": "^3.0.1",
     "react-test-renderer": "16.8.6",
     "redux-devtools": "3.5.0",
     "redux-mock-store": "1.2.3",
     "shx": "0.2.2",
-    "typescript": "^3.5.3",
```

Comment:
What was that doing here!

Reply:
There is a bug in `react-scripts` that complains about this being a missing peer dependency (https://github.com/facebook/create-react-app/issues/6834). Alex opted to remove the warnings by adding it as a dev dependency. I didn't have time to figure out what the right thing to do was for his first PR so allowed it in with a plan to promptly remove it again once I'd had a chance to look. The warnings aren't an issue so I've removed it to ensure there is no temptation for anyone to write any TypeScript.